content/practices/lean-experiments/index.md

```diff
@@ -0,0 +1,173 @@
+   {{< /callout >}}
+   
+   {{< callout >}}
+   Tip: If your team is not familiar with what makes good metrics, consider giving a primer or examples
+   {{< /callout >}}
+
+1. **Generate experiments**
+   
+   Distribute a sheet of paper to each team member and instruct them to:
+   
+   Copy the format of the Hypothesis & Experiment template
+   Write the leap-of-faith assumption as a falsifiable hypothesis
+   If there are more than 4 participants, consider breaking into teams to generate experiments
+   
+   {{< callout >}}
+   Tip: Print out your templates, email/share them, or create and share the digital workspace before the workshop. 
+   {{< /callout >}}
+      
+   {{< callout >}}
+   Tip: Prompt everyone to take 5 minutes individually to think of an experiment to test the hypothesis and associated measurable outcomes.
+   {{< /callout >}}
+
+1. **Share experiments**
+   
+   When time is up, have each team member (one by one) stick their sheet up on the wall and share the details of their experiment for group discussion.
+   
+   {{< callout >}}
+   Tip: Iteration on experiments from team feedback should be encouraged.
+   {{< /callout >}}
+
+1. **Select an experiment**
+   
+   Explain that the best experiment should require the least amount of effort for the maximum amount of learning. Have the team members silently dot vote (1 or 2 votes depending on group size and # of experiments) on the experiment they think is best to run.
+   
+   {{< callout >}}
+   Tip: If you think the experiment could use refinement, consider prompting the team to redesign the experiment. Assign different team members to create experiments for the same hypothesis that take 2 days with $200, 2 weeks with $2000, and 2 months with $20,000. This should help find the right size for the experiment and encourage ways to learn smaller.
+   {{< /callout >}}
+
+1. **Discuss how to evaluate your experiments**
+  
+   With the experiment selected, talk through how to evaluate it. Typically experiments result in one of three learning outcomes, each with an expected next step:
+  
+   - Uncertainty > test more & possibly iterate on experiment
+   - Validated > progress
+   - Invalidated > pivot
+  
+   For any of the outcomes, record the results of the experiment and planned next steps. A good resource to leverage is [The Learning Card](https://assets.strategyzer.com/assets/resources/the-learning-card.pdf) from Strategyzer.
+  
+   {{< callout >}}
+   Tip: It’s good practice to set a super fail condition on your experiment. This is a failure so bad that it could indicate the experiment itself has an issue. We should believe something to be true in order to test it, so an epic failure should be unlikely. 
+   {{< /callout >}}
+
+1. **Plan next steps & action items**
+   
+   Brainstorm and then assign actions for team members to take in order to execute the experiment.
+
```

Comment:
1. **Create experiment tracker 
Keep the not selected experiments in this session into your backlog. Regularly review the experiment backlog to keep executing the experiment that is relevant to validate your product direction

Reply:
Updated.